const stamFunc = () => {
    const error = new Error('this is error')
    throw error;
}